recipes/foursquare/datacube/init.sql

```diff
@@ -3,10 +3,14 @@ DECLARE
     _main boolean;
     _view boolean;
     _view_grouped boolean;
+    _view_weekly_zipcode boolean;
+    _view_daily_zipcode boolean;
 BEGIN
     SELECT 'foursquare_datacube.main' IN (SELECT table_schema||'.'||table_name FROM information_schema.tables) INTO _main;    
     SELECT 'foursquare_datacube.grouped_latest' IN (SELECT table_schema||'.'||table_name FROM information_schema.tables) INTO _view;
     SELECT 'foursquare_datacube.latest' IN (SELECT table_schema||'.'||table_name FROM information_schema.tables) INTO _view_grouped;
```

Comment:
Are the _view and _view_grouped tags backwards in these two lines?

Reply:
you are right